test/integration/tests/4045-no-run-benchmarks/Main.hs

```diff
@@ -0,0 +1,20 @@
+import Control.Monad (unless,when)
+import Data.List (isInfixOf)
+import StackTest
+
+main :: IO ()
+main = do
+    stack [defaultResolverArg, "clean"]
+    stack [defaultResolverArg, "build"]
+    stackOk ["test"] 
+    stackFail ["bench"] 
+    stackFail ["test", "--no-run-benchmarks"] 
+    putStrLn "DONE"
```

Comment:
I don't think we need the extra print in the test, probably is useful now while this is WIP though.

Reply:
👍